_plugins/auto_logs_markup.rb

```diff
@@ -38,6 +48,7 @@
     time    = line[0..TIME_SIZE]
     name    = IRC_NICK.match(line).to_s
     nick    = name.gsub(LT_GT, '').strip
+    color   = name_colors[nick] || (name_colors[nick] = colors[color_index += 1])
```

Comment:
Note that the colors array is currently 63 elements in size; if we have more than that many participants in a post, the color will become `nil` and fall back to the color (green) set in the `.log-nick` CSS class in all.sass. I don't think it's an issue, but we could check that `color_index < colors.size` and if true reset the index to 0, and/or ensure the color array is large enough (which also avoids anyone having the same color). For now I kept this to a one-liner in favor of a large color array.